cmd/backend/backend.go

```diff
@@ -40,9 +40,18 @@ type Options struct {
 	Backend BackendConfig
 }
 
+// BackendConfig defines the exposed backend configurations
 type BackendConfig struct {
-	Port       int    `env:"EPHEMERAL_BACKEND_PORT, default=8888"`
+	// Port defines the port used to listen to http requests sent to this service
+	Port int `env:"EPHEMERAL_BACKEND_PORT, default=8888"`
+	// Kubeconfig is an optional configuration to allow connecting to a k8s cluster
+	// remotelly
 	Kubeconfig string `env:"KUBECONFIG"`
+	// Namespace must point to the namespace where this backend service is running
+	Namespace string `env:"EPHEMERAL_BACKEND_NAMESPACE, required"`
+	// DefaultAccessDuration defines the default duration to be used when creating
+	// AccessRequests
+	DefaultAccessDuration time.Duration `env:"EPHEMERAL_BACKEND_DEFAULT_ACCESS_DURATION, default=4h"`
```

Comment:
Is this default for every change request?

Reply:
yes